react.gradle

```diff
@@ -102,20 +102,22 @@ def getHermesCommand = {
         }
     }
 
+    def hermescBin = Os.isFamily(Os.FAMILY_WINDOWS) ? 'hermesc.exe' : 'hermesc'
```

Comment:
Great stuff. Can I ask you to do the same logic here:

https://github.com/facebook/react-native/blob/c37f7195675df67d23c3c008ec5ab5fd7b8d0394/packages/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/utils/PathUtils.kt#L125-L161

Reply:
Sure! Thanks for the review BTW 🙂

Reply:
Done! I took a slightly different approach since the Kotlin was a bit different, but its still the same concept 😁